apollo-integration/build.gradle.kts

```diff
@@ -36,6 +36,7 @@ dependencies {
   add("implementation", "com.apollographql.apollo:apollo-rx2-support")
   add("implementation", "com.apollographql.apollo:apollo-coroutines-support")
   add("implementation", "com.apollographql.apollo:apollo-http-cache")
+  add("implementation", "com.apollographql.apollo:apollo-compiler")
```

Comment:
CI errors with duplicate class error. This may be the cause.

Reply:
yea, looks like it pulls both antlr and antlr-runtime at the same time...

Reply:
Would `compileOnly` work here? If not, you can provide a lambda block and call `exclude` function to exclude `antlr`. Not sure if that Gradle API is `Kotlin` friendly.

Reply:
I just removed the `antlr-runtime` dependency from apollo-compiler altogether 🤷‍♂: https://github.com/apollographql/apollo-android/pull/1859/commits/2c9672d966992dab93102af8ad6698c8c9724d58#diff-8d72e2bd6e0f687326024c0f1306de87L18

What's weird is that both `antlr` and `antlr-runtime` seem to include the same duplicate classes. I would have expected one to depend on the other. Anyways, it looks like it fixed the build.